src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+
+Web3 solves these problems by allowing you to control your digital identity with an Ethereum address and ENS profile. Using an Ethereum address provides a single login across platforms that is secure, censorship-resistant, and anonymous.
+
+<ButtonLink to="https://login.xyz/">
+  Sign-in with Ethereum
+</ButtonLink>
+
+### Native payments {#native-payments}
+
+Web2's payment infrastructure relies on banks and payment processors, excluding people without bank accounts or those who happen to live within the borders of the wrong country.
+Web3 uses tokens like ETH to send money directly in the browser and requires no trusted third party.
+
+<ButtonLink to="/eth/">
+  More on ETH
+</ButtonLink>
+
+## Web3 limitations {#web3-limitations}
+
+Despite the numerous benefits of Web3 in its current form, there are still many limitations that the ecosystem must address for it to flourish.
+
+### Accessibility {#accessibility}
+
+Web3 is less likely to be utilized in less-wealthy, developing nations due to high transaction fees. On Ethereum, these challenges are being solved right through network upgrades and layer 2 scaling solutions. The technology is ready, but we need higher levels of adoption on layer 2 to make Web3 accessible to everyone.
+
+<ButtonLink to="/layer-2/">
+  More on layer 2
+</ButtonLink>
+
+### User experience {#user-experience}
+
+The technical barrier to entry to using Web3 is currently too high. Users must comprehend security concerns, understand complex technical documentation, and navigate unintuitive user interfaces. Wallet providers, in particular, are working to solve this, but more progress is needed before Web3 gets adopted en masse.
+
+### Centralized infrastructure {#centralized-infrastructure}
+
+The Web3 ecosystem is young and quickly evolving. As a result, it currently depends mainly on centralized infrastructure (GitHub, Twitter, Discord, etc.). Many web3 companies are rushing to fill these gaps, but building high-quality, reliable infrastructure takes time.
+
+## A decentralized future {#decentralized-future}
+
+Web3 is a young and evolving ecosystem. Gavin Wood coined the term in 2014, but many of these ideas have only recently become a reality. In the last year alone, there has been a considerable surge in the interest in cryptocurrency, improvements to layer 2 scaling solutions, massive experiments with new forms of governance, and revolutions in digital identity.
+
+We are only at the beginning of creating a better internet with web3, but as we continue to improve the infrastructure that will support it, the future of the internet looks bright.
+
+## Further reading {#further-reading}
+
+Web3 isn’t rigidly defined. Various community participants have different perspectives on it. Here are a few of them:
```

Comment:
Potential articles to add here:
- https://onezero.medium.com/why-decentralization-matters-5e3f79f7638e